src/installer/tests/Directory.Build.targets

```diff
@@ -127,5 +127,19 @@
     </ItemGroup>
   </Target>
 
+  <!-- Properties used by runsettings.targets -->
+  <PropertyGroup>
+    <_withoutCategories>$(_withoutCategories);failing</_withoutCategories>
```

Comment:
That property isn't publicly exposed and not expected to be set. I think there's a `WithoutCategories` property instead in the tests or xunit targets / props.

Reply:
This was left over from a previous way I was trying to do this - removed.